msos/MixedStack.cs

```diff
@@ -312,5 +405,251 @@ public void PrintStackTrace(CommandExecutionContext context, uint index)
             var stackTrace = GetStackTrace(index);
             PrintStackTrace(context, stackTrace);
         }
+
+
+    }
+    class UnifiedUnManagedThread : UnifiedThread
+    {
+        public UnifiedUnManagedThread(ThreadInfo info, List<UnifiedStackFrame> unmanagedStack, List<UnifiedBlockingObject> blockingObjects) : base(info)
+        {
+            BlockingObjects = blockingObjects;
+            StackTrace = unmanagedStack;
+        }
+    }
+
+    public enum UnifiedHandleType
+    {
+        Handle, CriticalSection
+    }
+
+    public class UnifiedHandle
+    {
+        public UnifiedHandle(ulong uid, UnifiedHandleType unifiedType = UnifiedHandleType.Handle, string type = null, string objectName = null)
```

Comment:
What does `uid` mean here, and what does the `Id` property mean? Is it the handle value? Then it should probably be called `HandleValue` or `Value`.